flake.nix

```diff
@@ -10,7 +10,7 @@
       let
         pkgs = import nixpkgs {
           inherit system;
-          overlays = [ typelevel-nix.overlay ];
+          overlays = [ typelevel-nix.overlays.default ];
```

Comment:
This change comes from https://github.com/typelevel/fs2/pull/3547. I will rebase when it get's merged.